pkgs/servers/openafs/1.8/patches/do_settimeofday.patch

```diff
@@ -0,0 +1,60 @@
+commit aa80f892ec39e2984818090a6bb2047430836ee2
```

Comment:
This looks exactly like
http://git.openafs.org/?p=openafs.git;a=patch;h=aa80f892ec39e2984818090a6bb2047430836ee2
So we should be able to use fetchpatch after all.

Reply:
It seems my problem with fetchpatch was assuming `nix-prefetch-url` would give me the right hashes. I've switched to fetchpatch.

Reply:
Yes, fetchpatch will apply some extra filtering to the patches, so the hash from the fixed-output derivation will often not match the original patches hash.